pkg/vm/evm_settings.go

```diff
@@ -4,11 +4,15 @@
 package vm
 
 import (
+	_ "embed"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
+//go:embed evm_debug_config.json
+var EvmDebugConfig []byte
```

Comment:
Nit: To follow [standard conventions](https://go.dev/doc/effective_go), abbreviations should all be the same case. Since we want this exported here, I think we should have it be `EVMDebugConfig` to conform.